athlinks-ontology.ttl

```diff
@@ -86,6 +86,24 @@
 	rdfs:comment "The numeric order of an object among siblings of the same property or type."@en ;
 	rdfs:label "sibling ordering"@en .
 
+:steps a owl:DatatypeProperty ;
+  rdfs:domain :AthleticEventCompetitor ;
+  rdfs:range xsd:integer;
+  rdfs:comment "The number of steps an athlete took in an athletic event"@en ;
+  rdfs:label "steps"@en .
+
+:startTime a owl:DatatypeProperty ;
```

Comment:
I think these should be on an action, not here.

Reply:
Why is that? I think actions just add unnecessary complexity.

Reply:
I think we need to have steps on a representation of Intervals (Actions), such that for any given interval we could ask for the steps taken.
e.g

--  Full Interval - 3745 Steps
--  5k - 1623 Steps
-- 1 mile - 580 Steps

Reply:
Yes, @Astn is right regarding intervals. All but the simplest races have multiple intervals or sub-courses, and we need to track results for each one. This was the intent of modeling these results using schema.org's `Action` concept. It is an excellent fit for a race result. An `Action` has an `agent` (the competitor), a `location` (the course or sub-course),  a `startTime` and `endTime` and a `result` property, which in this case can correspond to a time result or a steps result. The current ontology has a concept`athlinks:AthleticEventAction` which is a sub-class of `schema:ExerciseAction`, and which has some specialized sub properties like `distance`, `exerciseCourse` and `sportsEvent` that are event more specific than the general `schema:Action` properties.